### ISSUE TITLE
why not support obj in container?

### ISSUE THREAD

**Issue by WankkoRee** [samme/phaser-plugin-debug-draw#13]
i try this and all thing is right.

![image](https://user-images.githubusercontent.com/21287731/235440811-b93664e1-dc67-4e8d-9d58-e300699d7ede.png)

and `obj.list` is also okay with `container`

**Comment by samme** [comment]
But unlike with layers, it would need the transformed position of the children:

    const { tx, ty } = gameObject.getWorldTransformMatrix();

I guess it could be added.

**Comment by WankkoRee** [comment]
yes, I try to replace `obj.x, obj.y` to `{ tx, ty } = gameObject.getWorldTransformMatrix()`, it works when container has been set x or y.

![image](https://user-images.githubusercontent.com/21287731/235479497-4a82bc28-bf30-457f-974f-71b6d7aca46e.png)
